css/src/components/component.md

```diff
@@ -0,0 +1,8 @@
+#### Components
+
+Create new component when you have a complex layout that requires many helper classes to recreate. Keep following in mind while creating a New component .
```

Comment:
"Create a new component ......... recreate. Keep the following in mind while creating a new component."